pages/docs/reason-compiler/latest/decorators.mdx

```diff
@@ -525,7 +525,16 @@ var myBook = new Book();
 
 ### `[@bs.deriving accessors]`
 
-Reason records are represented as arrays in JS.
+Reason records were represented as arrays in JS in BuckleScript versions < v7. `bs.deriving accessors` makes it more convenient to access records when you or your library consumers need to be on an older BuckleScript version.
```

Comment:
I feel like we shouldn't talk too much about the backwards compatibility aspects and write more about what the `accessors` feature is about.

Reply:
So just ``bs.deriving accessors` makes it more convenient to access records`? Because the rest was already mentioned, I mainly wanted to get rid of the `Reason records are represented as arrays in JS`.